packages/cli/src/utils/systems.ts

```diff
@@ -0,0 +1,192 @@
+import chalk from "chalk";
+import { Contract, ethers } from "ethers";
+import { TransactionReceipt, TransactionResponse } from "@ethersproject/providers";
+import { ParamType } from "ethers/lib/utils.js";
+import { tableIdToHex } from "@latticexyz/common";
+import { getContractData, TxConfig, fastTxExecute } from "./txHelpers";
+
+type SystemsConfig = Record<
+  string,
+  {
+    name: string;
+    registerFunctionSelectors: boolean;
+    openAccess: boolean;
+    accessListAddresses: string[];
+    accessListSystems: string[];
+  }
+>;
+
+interface FunctionSignature {
+  functionName: string;
+  functionArgs: string;
+}
+
+export async function registerSystems(
```

Comment:
same here, feels like it would be nice to have a small `registerSystem` and `registerFunctionSelector` util and then use it to create a small `registerSystems` util